daemon/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "daemon"
```

Comment:
I guess only one daemon means you will be having two binaries within that crate?

Reply:
that was the idea so far, yes